indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexTuningConfig.java

```diff
@@ -105,7 +105,7 @@ public ParallelIndexTuningConfig(
       @JsonProperty("maxRowsInMemory") @Nullable Integer maxRowsInMemory,
       @JsonProperty("maxBytesInMemory") @Nullable Long maxBytesInMemory,
       @JsonProperty("maxTotalRows") @Deprecated @Nullable Long maxTotalRows,
-      @JsonProperty("numShards") @Nullable Integer numShards,
+      @JsonProperty("numShards") @Deprecated @Nullable Integer numShards,
```

Comment:
do we need to update any docs if `numShards` is deprecated now.

Reply:
It's already mentioned in the doc. I just added a missing annotation.

Reply:
ah cool, thanks